.github/workflows/validate.yml

```diff
@@ -40,7 +40,7 @@ jobs:
           sudo apt -qy install libseccomp-dev
       - uses: golangci/golangci-lint-action@v7
         with:
-          version: v2.0
+          version: v2.1
```

Comment:
How about deleting this version definition?

Reply:
Assuming that if the version is not set, the latest one will be used, I don't think it's a good idea, because this will mean that over time our CI will break on its own (as newer golangci-lint will update or add more linters which may result in linter warnings).

By pinning the version, we avoid that breakage. When we update the version, we check that everything works fine, or fix the new issues found. In the meantime, the CI works.

It's basically the same argument as in commit b24fc9d2

Reply:
Note though that I don't specify the patch version, meaning the latest patch version will be used (for example, when I specify v2.1, it will use v2.1.5 as of today). This is because I _hope_ that the patch version bump won't introduce any breakage (while the minor version bump, such as switching from v2.2 to v2.3, might).

Reply:
Make sense, thanks your explanation.